tests/issue55.md

```diff
@@ -0,0 +1,74 @@
+---
+plot-configuration: tests/fixtures/.verbose-config.yml
+---
+
+# Example Asymptote plot
+
+```{.asy}
```

Comment:
The default PNG format isn't supported; please change this to:

````{markdown
```{.asy format=pdf}
````

Reply:
Found reason for PNG issues, and added support for default PNG format.
However, I would recommend using PDF or EPS as default output formats, since they are vector formats.